src/Display.tsx

```diff
@@ -109,31 +110,20 @@ const ShelterDetail: React.FC<ShelterDetailProps> = props => {
   return <FlexItem>{shelterName}</FlexItem>;
 };
 
-interface ShelterDetailProps {
-  id: string;
-}
-const ShelterDetail: React.FC<ShelterDetailProps> = props => {
-  const [shelterName, setShelterName] = useState("");
-
-  fetch(process.env.REACT_APP_BACKEND_URL + `/find/shelter/${props.id}`)
-    .then(response => response.json())
-    .then(jsonData => {
-      setShelterName(jsonData.map.shelter.map.name);
-    });
-  return (
-    <>
-      <GridItem span={3}>Designated Shelter: </GridItem>
-      <GridItem span={9}>{shelterName}</GridItem>
-    </>
-  );
-};
```

Comment:
Why is this removed?

Reply:
This was replaced by the FlexItems. And I had errors when cloned with map-display branch. The ShelterDetail was repeated twice. So, committed the latest code by removing the above code.